File: C#/Basic/array sum.cs
using System;

class Program
{
    static void Main()
    {
        int[] numbers = { 1, 2, 3, 4, 5 };
        int sum = 0;

        foreach (int number in numbers)
        {
            sum += number;
        }

        Console.WriteLine("Sum of array elements: " + sum);
    }
}

File: C++/Basic/array sum.cpp
#include <iostream>

int main() {
    int numbers[] = {1, 2, 3, 4, 5};
    int sum = 0;

    for (int num : numbers) {
        sum += num;
    }

    std::cout << "Sum of array elements: " << sum << std::endl;
    return 0;
}
